CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- Add `from` based on fowarder header in log context
```

Comment:
Similar question to the one in IOTAs repos... is there any document about logs in this repo that should be modified based on this PR?

Reply:
I've found this doc https://github.com/telefonicaid/fiware-pep-steelskin/blob/b766e6c2013422195d66cf07c5ce320c85d0ca49/operations.md#--logs but no format is described

Reply:
NTC